### PR TITLE
Show a simplified view for animal details

### PR DESCRIPTION
**Pull request by JasoonS**

### CODE REVIEW

**Comment by JasoonS** [comment]
When user isn't linked on on Github.

When user isn't 3box verified:
image

Expert toggle - user is 3box verified
Peek 2020-11-17 15-14

File: src/harberger-lib/components/Info.re [hunk 1]
```diff
@@ -1,5 +1,342 @@
...
+               clearAndPush({j|/#user/$currentPatron|j});
+             }}>
+             displayNameStr->restr
+           </a>
+           {(
+              " has been the guardian of "
+              ++ tokenName
+              ++ " for "
+              ++ daysHeldFloat->Js.Float.toFixed
+              ++ " days"
```
Comment (DenhamPreen) [inline src/harberger-lib/components/Info.re]: Add a space after days

File: src/harberger-lib/components/Info.re [hunk 2]
```diff
@@ -1,5 +1,342 @@
...
+            )
+            ->React.string}
+           {switch (definiteTime) {
+            | Some(date) =>
+              <>
+                "and has enough deposit to last "->React.string
+                <CountDown endDateMoment=date />
+                {j| - remember to keep topping up that deposit 😉.|j}
```
Comment (DenhamPreen) [inline src/harberger-lib/components/Info.re:299]: Have we used emoji's before? It shows as a different char in the ui for me but I feel like we did have a way to add emojis before. Not a biggy

File: src/harberger-lib/components/Info.re [hunk 3]
```diff
@@ -75,182 +411,52 @@ let make = (~chain, ~tokenId: TokenId.t) => {
...
-       </p>;
-     | None => React.null
-     }}
-  </React.Fragment>;
+  let currentPriceWei = QlHooks.usePrice(~chain, tokenId);
+  let priceString =
+    switch (currentPriceWei) {
+    | Foreclosed(price)
+    | Price(price) =>
+      price->Web3Utils.fromWeiBNToEthPrecision(~digits=4) ++ " USD"
```
Comment (DenhamPreen) [inline src/harberger-lib/components/Info.re]: I think it's better to show 2 digits here, 4 makes it seem more than it is.

**Comment by DenhamPreen** [comment]
Jason this is all awesome! I like the clear use of text. 

I don't think its that clear that the expert toggle changes the copy and there is a bug that if you come from the wildcards explorer view that the toggle doesn't show. 
(To replicated this go to -> https://ui-git-simple-details.jasoons.vercel.app/  -> 'click' VIEW WILDCARDS -> Click on an animal -> You will see there is no toggle, the toggle shows if you come directly from the home page)
I would recommend simply adding the same toggle in the top right of the details tab, this emphasizes a bit more that its linked to this component.

My last points but not biggies 
- is to make the links slightly lighter in colour (or make the underline permanent) 
- Change the address truncation to the middle rather than the end  ~0x0cacc61...~    ->     *0x0ca...1ad23* 

Awesome stuff

File: src/harberger-lib/components/Info.re (hunk 2 above)
Comment (JasoonS) [reply src/harberger-lib/components/Info.re:299]: Firefox and chrome have different emojis (of the same emojis).

I don't think we have used emojis in wildcards before, only other places

File: src/harberger-lib/components/Info.re (hunk 3 above)
Comment (JasoonS) [reply src/harberger-lib/components/Info.re]: Yes, totally agree. And it shouldn't be hardcoded to USD either...

**Comment by JasoonS** [comment]
>I don't think its that clear that the expert toggle changes the copy and there is a bug that if you come from the wildcards explorer view that the toggle doesn't show.

Yes, you're right, that bug used to be a feature because that page never used to have a need for the toggle. But agree, adding it directly on the details page would help a lot!

This was one of Rios feature requests, so credit to her :)

**Pull request merged**